htdocs/website/index.php

```diff
@@ -2155,6 +2155,65 @@
 	}
 }
 
+$form = new Form($db);
+$formadmin = new FormAdmin($db);
+$formwebsite = new FormWebsite($db);
+$formother = new FormOther($db);
+$domainname = '0.0.0.0:8080';
+$tempdir = $conf->website->dir_output.'/'.$websitekey.'/';
+
+// Confirm generation of website sitemaps
+if($action == 'confirmgeneratesitemaps'){
+	$formconfirm = $form->formconfirm($_SERVER["PHP_SELF"].'?website='.$website->ref, $langs->trans('ConfirmSitemapsCreation'), $langs->trans('ConfirmGenerateSitemaps', $object->ref),'generatesitemaps', '', "yes", 1);
```

Comment:
This part of code, like the declaration of $form are code related to "presentation". So, to respect MVC rules, we should not have it before the comment

```
/*
 * View
 */
```

The code should be placed back again after the View separator